### ISSUE TITLE
Allow parameter -p to attach to running processes

### ISSUE THREAD

**Issue by didierm** [jonasdn/nsntrace#19]
Akin to strace : 
`-p pid         trace process with process id PID`

**Comment by jonasdn** [comment]
I do not think this is possible atm. I do not know of a way of changing a running process network namespace.
